telethon/extensions/html.py

```diff
@@ -195,8 +197,14 @@ def unparse(text: str, entities: Iterable[TypeMessageEntity], _offset: int = 0,
             html.append('<u>{}</u>'.format(entity_text))
         elif entity_type == MessageEntityStrike:
             html.append('<del>{}</del>'.format(entity_text))
+        elif entity_type == MessageEntityStrike:
+            html.append('<s>{}</s>'.format(entity_text))
+        elif entity_type == MessageEntityStrike:
```

Comment:
sorry if i'm missing something but isn't this the same condition as above?

Reply:
> sorry if i'm missing something but isn't this the same condition as above?



> sorry if i'm missing something but isn't this the same condition as above?

Yes is same, because an alias for ~strike~.
- For html use `<del>`, `<s>`, or `<strike>`.
- For markdown use `~~`

Reference:
- [Allowed entities](https://core.telegram.org/api/entities#allowed-entities)

Reply:
But as @hughbe points out, this would never run, because the one above just ran.